pages/lists/addplanttoplace.js

```diff
@@ -1,19 +1,31 @@
 import { plants } from "@/db/db";
-import { StyledButton, StyledLink } from "@/components/StyledButton";
+import { StyledButton, StyledInactiveButton } from "@/components/StyledButton";
 import {
   StyledCard,
   StyledCardList,
   StyledCardListItem,
   StyledContentContainer,
 } from "@/components/Card";
 import { useRouter } from "next/router";
+import { StyledMain } from "@/components/Main";
+import StyledNavigation from "@/components/Navigation";
+import { StyledHeader } from "@/components/Header";
 
 const plantsSorted = [...plants].sort((a, b) => (a.name > b.name ? 1 : -1));
 
 export default function AddPlantToPlaceList({ places, setPlaces }) {
   const router = useRouter();
   const site = router.query;
 
+  if (!site.id) {
+    return null;
+  }
+
+  const place = places[site.id - 1];
+  const freePlaces = place.capacity - place.used;
+  const addButton = freePlaces <= 0 ? false : true;
+  const minusButton = freePlaces >= place.capacity ? false : true;
```

Comment:
I would call `addButton` and `minusButton`: `showAddButton` and `showMinusButton`, to give a better boolean feeling ^^